### DISCUSSION TITLE
QSlideItem: Support multiple buttons on each side ("partial slide")

### DISCUSSION THREAD

**Discussion by hognevevle** [quasarframework/quasar#16744] [Ideas / Proposals]
Currently you are limited to one item on each side (left / right). Sliding will automatically slide "the entire item". 

Would be great to be able to slide only partially, which would allow you to select one of several options (e.g. two or more buttons) -- or continue the sliding motion to select the outermost option. 

Essentially what I'm looking for is what Ionic offers in their `ion-item-sliding` (see [Expandable options](https://ionicframework.com/docs/api/item-sliding#expandable-options)).
 
![CleanShot 2024-01-04 at 12 13 33](https://github.com/quasarframework/quasar/assets/2721188/65a495a9-cb9a-41fb-bdee-ae1fcb6a4410)

**Comment by conx13** [comment]
hey. could we expect such a solution in the future?
